raiden/network/rpc/client.py

```diff
@@ -320,12 +325,12 @@ def deploy_solidity_contract(
                     raise RuntimeError('Contract address has no code, check gas usage.')
 
             hex_bytecode = solidity_resolve_symbols(contract['bin'], libraries)
-            bytecode = unhexlify(hex_bytecode)
+            bytecode = decode_hex(hex_bytecode)
 
             contract['bin'] = bytecode
 
         if isinstance(contract['bin'], str):
```

Comment:
@jomuel I propose changing this line to `if not symbols:`, since the type of `contract['bin']` can only be an `str` if there are no unresolved symbols. What do you think?